service/history/tasks/category.go

```diff
@@ -130,6 +131,10 @@ func (c Category) Type() CategoryType {
 	return c.cType
 }
 
+func (c Category) MarshalText() (text []byte, err error) {
+	return []byte(fmt.Sprintf("[id:%d][type:%s][name:%s]", c.id, c.cType, c.name)), nil
+}
```

Comment:
This is needed to be able to serialize `Tasks map[tasks.Category][]tasks.Task`

Reply:
Same comment as I left in previous PR: `c.name` should be good enough because it is in fact task key. Other fields adds more confusion than clarity.